DeveloperGuide.rst

```diff
@@ -88,10 +88,27 @@ Run the following command::
 Mac OS X
 ^^^^^^^^
 
-**Note:** These instructions are not complete due to not having a bare OS X machine to test on. However,
-they should give you some hints as to how to get an OS X machine set up to develop pepys-import.
+The easiest way to get the relevant libraries working on a M1 MacOS machine is to use the conda Python package manager which will install from the
+packages provided by conda-forge. This is different to the standard pip/virtualenv process, but works quite nicely alongside it.
+The conda package manager can be quite slow, so the instructions show how to install mamba instead, which is a faster fork.
 
-**TODO - Complete instructions here**
+These instructions are for Macs with a M1 (ARM) processor. A similar approach will work for x86 (Intel) Macs, but you will need
+to ensure that you install the Intel version of miniconda.
+
+1. Install the ARM version of mambaforge, which is available `here <https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-MacOSX-arm64.sh>`_, following all the instructions to the end (including setting up integration with your shell)
+
+2. Create a new environment (like conda's equivalent of a virtualenv) by running: ``mamba create -n pepys python=3.9 libspatialite``. This will
+   install Python 3.9 and libspatialite in a new environment called ``pepys``.
+
+3. Activate the environment by running ``mamba activate pepys``
+
+4. Install the other Pepys dependencies by running ``pip install -r requirements.txt -r requirements_dev.txt``
```

Comment:
This step requires `pg_config`.  We don't install `postgres.app` until later on.  Did you have another `pg_config` present?

Otherwise, I think we need to install `postgres.app`, include a step put `contents\latest\bin` on the path, then install the dependencies.